components/markdown_image.jsx

```diff
@@ -50,14 +53,34 @@ export default class MarkdownImage extends React.PureComponent {
         this.setState({loadFailed: true});
     }
 
+    handleImageLoaded = ({height, width}) => {
```

Comment:
How is this function being called?

Reply:
Oops, sorry 😔. Definitely a mistake. It was meant to be passed down as a prop to `SizeAwareImage`. However, I've realized that there is one variation of the issue Andrew reported (needing to refresh to see correct image in channel header after a broken one) that was not fixed yet. I've adopted a different approach for the fix that no longer needs this function. Thanks for this!